crates/modelardb_manager/src/cluster.rs

```diff
@@ -137,84 +137,42 @@ impl Cluster {
-    /// `table_name`. If the table was successfully dropped for each node, return [`Ok`], otherwise
-    /// return [`ModelarDbManagerError`].
-    pub async fn drop_table(&self, table_name: &str, key: &MetadataValue<Ascii>) -> Result<()> {
-        let action = Action {
-            r#type: "DropTable".to_owned(),
-            body: table_name.to_owned().into(),
-        };
-
-        self.execute_cluster_action(action, key).await
-    }
-
-    /// For each node in the cluster, use the `TruncateTable` action to truncate the table given by
-    /// `table_name`. If the table was successfully truncated for each node, return [`Ok`], otherwise
-    /// return [`ModelarDbManagerError`].
-    pub async fn truncate_table(&self, table_name: &str, key: &MetadataValue<Ascii>) -> Result<()> {
-        let action = Action {
-            r#type: "TruncateTable".to_owned(),
-            body: table_name.to_owned().into(),
-        };
-
-        self.execute_cluster_action(action, key).await
-    }
-
-    /// For each node in the cluster, execute the given `action` with the given `key`. If the action
-    /// was successfully executed for each node, return [`Ok`], otherwise return [`ModelarDbManagerError`].
-    async fn execute_cluster_action(
-        &self,
-        action: Action,
-        key: &MetadataValue<Ascii>,
-    ) -> Result<()> {
-        let mut action_futures: FuturesUnordered<_> = self
+    pub async fn cluster_do_get(&self, sql: &str, key: &MetadataValue<Ascii>) -> Result<()> {
+        let mut do_get_futures: FuturesUnordered<_> = self
             .nodes
             .iter()
-            .map(|node| self.connect_and_do_action(&node.url, action.clone(), key))
+            .map(|node| self.connect_and_do_get(&node.url, sql, key))
             .collect();
 
         // TODO: Fix issue where we return immediately if we encounter an error. If it is a
         //       connection error, we either need to retry later or remove the node.
-        // Run the futures concurrently and log when the action has been executed on each node.
-        while let Some(result) = action_futures.next().await {
-            info!(
-                "Executed action `{}` on node with url '{}'.",
-                action.r#type, result?
-            );
+        // Run the futures concurrently and log when the statement has been executed on each node.
+        while let Some(result) = do_get_futures.next().await {
+            info!("Executed statement `{sql}` on node with url '{}'.", result?);
         }
 
         Ok(())
     }
 
-    /// Connect to the Apache Arrow flight client given by `url` and make a request to do `action`.
-    /// If the action was successfully executed, return the url of the node, otherwise return
-    /// [`ModelarDbManagerError`].
-    async fn connect_and_do_action(
+    /// Connect to the Apache Arrow flight client given by `url` and execute the given `sql`
```

Comment:
Is the flight client not used to connect to the instance of `modelardbd` at `url` instead of a Apache Arrow flight client? Maybe rephrase the comment.